### ISSUE TITLE
`MaxLineLength` `excludeRawStrings` false negative

### ISSUE THREAD

**Issue by GrzegorzBobryk** [detekt/detekt#7555]
## Expected Behavior

After setting `excludeRawStrings` on `true` in `MaxLineLength` linter should ignore raw strings.

## Observed Behavior

As shown [here](https://github.com/GrzegorzBobryk/ViewBindingDelegate/pull/20/files). If we create raw string that is single line detekt will see error in it. Moving only closing quote signs to new line is making it pass lint check.

Logs [here](https://github.com/GrzegorzBobryk/ViewBindingDelegate/actions/runs/10442354185/job/28914579007?pr=20).
```shell
/home/runner/work/ViewBindingDelegate/ViewBindingDelegate/sample/src/main/kotlin/pl/beavercoding/viewbindingdelegate/ExampleFragment3.kt:30:1: Exceeded max line length (120) [MaximumLineLength]
/home/runner/work/ViewBindingDelegate/ViewBindingDelegate/sample/src/main/kotlin/pl/beavercoding/viewbindingdelegate/ExampleFragment3.kt:30:1: Line detected, which is longer than the defined maximum line length in the code style. [MaxLineLength]
```

## Steps to Reproduce

- Set `excludeRawStrings: true` in `MaxLineLength`;
- Add some long raw single line string that should brake linter if it was a normal line. For example:
`"""C:\Repository\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\test\some_file.pdf"""`
![obraz](https://github.com/user-attachments/assets/534cac43-e88b-4a15-b7c6-b74c0e6eb71b)
- Run detekt;

## Context

I want to ignore all raw strings. I can create one with special characters that do not need to be escaped. For example very long path to file.

## Your Environment

* Version of detekt used: 1.23.6
* Version of Gradle used (if applicable): 8.7
* Gradle scan link (add `--scan` option when running the gradle task):
* Operating System and version: Windows 11
* Link to your project (if it's a public repository): included above
 [pr with bug example](https://github.com/GrzegorzBobryk/ViewBindingDelegate/pull/20/files)
 [logs from CI/CD](https://github.com/GrzegorzBobryk/ViewBindingDelegate/actions/runs/10442354185/job/28914579007?pr=20)

**Comment by 3flex** [comment]
I understand that option is for lines that only have a raw string and nothing else on that line, not for lines that happen to contain a raw string, so this seems to be working as intended.

If the line contains a raw string and other elements then the raw string should go on its own line which would stop this being reported.

The documentation could be improved here.

**Comment by GrzegorzBobryk** [comment]
As you see here declaration is moved to separate line, so raw string have line for himself. We do not have any other elements except raw string. But still it is getting reported.

**Comment by GrzegorzBobryk** [comment]
This will get reported:
![obraz](https://github.com/user-attachments/assets/51781b85-f6fc-4c05-bbbf-543a44b5b066)
This will be also reported:
![obraz](https://github.com/user-attachments/assets/d107f64c-baa5-499c-8a13-688ef7c24537)
But this will **NOT** be reported:
![obraz](https://github.com/user-attachments/assets/d2bdb3d9-41f3-4671-a776-27e229733440)

I would personally assume that if raw string is only element in line and opening and closing quotes are in the same line this should not be an error. But I will gladly hear your opinion on that.

**Comment by 3flex** [comment]
> I would personally assume that if raw string is only element in line and opening and closing quotes are in the same line this should not be an error. But I will gladly hear your opinion on that.

You're right, I don't think this should be an error. Thanks for confirming!

**Comment by liz-mills** [comment]
I also run into this issue often, and would ideally like to have my single-line long strings without having to putting a suppress above every one of them... Just adding to this convo so you know that this is wanted by others too!

**Comment by BraisGabin** [comment]
This issue is ready to grab by anyone willing to contribute.
